specification/databricks/resource-manager/Microsoft.Databricks/preview/2021-04-01-preview/databricks.json

```diff
@@ -652,6 +652,52 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Databricks/workspaces/{workspaceName}/outboundNetworkDependenciesEndpoints": {
+      "get": {
+        "tags": [
+          "GET"
+        ],
+        "summary": "Gets a list of egress endpoints (network endpoints of all outbound dependencies) in the specified Workspace.",
+        "description": "Gets a list of egress endpoints (network endpoints of all outbound dependencies) in the specified Workspace. The operation returns properties of each egress endpoint.",
+        "operationId": "OutboundNetworkDependenciesEndpoints_List",
+        "x-ms-examples": {
+          "List OutboundNetworkDependenciesEndpoints by Workspace": {
+            "$ref": "./examples/OutboundNetworkDependenciesEndpointsList.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
```

Comment:
Change from OK to "TThe response contains a list of outbound network dependencies."

Reply:
Changed Description